sources/packages/backend/apps/api/src/testHelpers/auth/institution-auth-helpers.ts

```diff
@@ -140,3 +140,9 @@ export async function authorizeUserTokenForLocation(
     InstitutionUserTypes.user,
   );
 }
+
+export const INSTITUTION_BC_PUBLIC_ERROR_MESSAGE =
```

Comment:
I am ok to keep those here. I was just having second thoughts if we should move or not to a constant file.
Either way, I would be good with 😉